src/lib/falcor/routes/articles/by-id.js

```diff
@@ -0,0 +1,27 @@
+import falcor from 'falcor';
+
+import { articleQuery } from 'lib/db';
+
+const $ref = falcor.Model.ref;
+
+export default [
+  {
+    /* 
+     * Get custom article slug from MariaDB and create ref
+     * It should be noted that this is the Falcor correct way
+     * to have multiple routes to access the same data, however
+     * it is inefficient from a SQL perspective since multiple
+     * SQL queries are used to access the same data when we are
+     * querying by ID. 
+     */
+    route: "articles['byId'][{keys:ids}]", // eslint-disable-line max-len
+    get: async pathSet => {
```

Comment:
Hmmm, I see what's hurting you more now, will be interesting to talk about it tomorrow